Misc/NEWS.d/next/Core and Builtins/2020-03-15-07-40-11.bpo-1635741.kL7c0f.rst

```diff
@@ -0,0 +1 @@
+Port _ctypes_test extension module to multiphase initialization (:pep:`489`).
```

Comment:
Please remove the NEWS entry. It's a test module, I don't think that it's worth it to document it for end users.

Reply:
got it, have been removed.